ui/src/i18n/fi/fi.json

```diff
@@ -12,7 +12,10 @@
         "publication": {
             "generic": "Julkaisu epäonnistui",
             "lock-obtain-failed": "Toinen julkaisu käynnissä - yritä hetken kuluttua uudelleen",
-            "validation-failed": "Julkaisuvalidointi epäonnistui"
+            "validation-failed": "Julkaisuvalidointi epäonnistui",
+            "duplicate-name-on-location-track": "Julkaisussa on useammalla sijaintiraiteella sama nimi {{0}} (ratanumerolla {{1}})",
+            "error.publication.duplicate-name-on.SWITCH": "Julkaisussa on useammalla vaihteella sama nimi {{0}}",
+            "error.publication.duplicate-name-on.TRACK_NUMBER": "Julkaisussa on useammalla radalla sama ratanumero {{0}}"
```

Comment:
Näissä nämä käännösavaimet ovat edelleen vähän outoja. Tuleekos näistä nyt kokonaisuudessaan arvoiksi `error.publication.error.publication.duplicate-name-on.TYYPPI`? Enivei tuosta `duplikate-name-on`:ista voisi tehdä ihan objektin tänne `publication`:in sisälle ja laittaa nuo pari arvoa sinne, vähän samaan tyyliin kuin tuossa ylempänä on tuo `integration-not-configured`

Reply:
Niijjo, olin katsonut vaan että eihän niissä tuo epäsymmetria ole yhtään outoa (vähän rumaa vain), mutta olihan niissä muutakin väärin :sweat_smile: